src/schedule/2021-02-08-add-calendar-links-to-11ty-with-computed-data.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Add Calendar Links to 11ty with Computed Data
+type: Mob Job
+date: 2021-02-08T01:00:00-05:00
```

Comment:
Thank you for submitting an event, @roman-bytes!

This is currently showing up as 10 **pm** PST - I'm guessing you want 10 **am** PST?

```suggestion
date: 2021-02-08T13:00:00-05:00
```

Come to think of it, we should probably make it a little clearer that the CLI is expecting a 24-hour format 😅

Reply:
whoops, lets not do this at 1am. Should be fixed now.